gum/backend-arm64/gumstalker-arm64.c

```diff
@@ -2079,10 +2130,12 @@ gum_exec_block_write_event_submit_code (GumExecBlock * block,
     gum_arm64_writer_put_cbnz_reg_label (cw, ARM64_REG_X14, beach_label);
 
     gum_arm64_writer_put_push_all_x_registers (cw);
+    gum_arm64_writer_put_push_all_q_registers (cw);
```

Comment:
We don't save registers here on x86, because that's what the opened prolog has already done. Is this a bug?